src/Test/DatabaseResetter.php

```diff
@@ -34,11 +34,33 @@ public static function isDAMADoctrineTestBundleEnabled(): bool
         return \class_exists(StaticDriver::class) && StaticDriver::isKeepStaticConnections();
     }
 
+    public static function isResetUsingMigrations(): bool
```

Comment:
```suggestion
    private static function isResetUsingMigrations(): bool
```

I don't think there is a reason to use this method outside this class (and makes it easier to modify this in future)